app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<main>
+<section class="spotlight">
+  <h2 class="spotlight__header">
+    <span class="spotlight__header--prefix">Media Spotlight:</span>
+    <% unless Work.spotlight.nil?  %>
+      <%= link_to Work.spotlight.title, work_path(Work.spotlight.id), class: "spotlight__link-to" %> by <%= Work.spotlight.creator %>
+      </h2>
+      <p class="spotlight__description">
+        <%= Work.spotlight.votes.count %> votes - <%= Work.spotlight.description %>
+      </p>
+    <% end  %>
```

Comment:
Because of where your `unless` is placed you wind up opening but never closing the `h2` if the spotlight is `nil`.